### ISSUE TITLE
Computing likelihood

### ISSUE THREAD

**Issue by ex3ploiter** [cvignac/DiGress#62]
Hello. Is there any option to compute the likelihood for the given graph in this model?

**Comment by cvignac** [comment]
Yes, the NLL for validation and test graphs is reported, you just have to replace the val/test datasets.
